<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu_image_name_input</class>
 <widget class="QMainWindow" name="menu_image_name_input">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>860</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1095</width>
    <height>860</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>510</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mbttonArea">
     <item>
      <widget class="QPushButton" name="mbtn_add">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>추가</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mbtn_mod">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>수정</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mbtn_del">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>삭제</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>40</y>
      <width>251</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="check_category_combobox">
       <item>
        <property name="text">
         <string>선택</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>커피</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>음료</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>차</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>플렛치노</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>쉐이크&amp;에이드</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="id_label">
         <property name="text">
          <string>메뉴 ID   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="check_menu_id">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>메뉴 ID를 입력해주세요</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="name_label">
         <property name="text">
          <string>메뉴 이름</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="check_menu_name">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>메뉴의 이름을 입력해주세요</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="price_label">
         <property name="text">
          <string>메뉴 가격</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="check_menu_price">
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>메뉴의 가격을 입력해주세요</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>조회 페이지</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>480</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>상태 페이지</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mbtn_check">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>370</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>조회</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>731</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblMenu2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>321</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>540</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="category_combobox">
       <item>
        <property name="text">
         <string>커피</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>음료</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>차</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>플렛치노</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>쉐이크&amp;에이드</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="menu_id_input">
       <property name="placeholderText">
        <string>⭐ 메뉴 ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="menu_name_input">
       <property name="placeholderText">
        <string>⭐ 메뉴 이름</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="menu_price_input">
       <property name="placeholderText">
        <string>⭐ 메뉴 가격</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="menu_image_input">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>이미지</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="menu_info_input">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>690</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>⭐ 메뉴 설명</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="menu_image_input_name">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>510</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>⭐ 이미지 이름</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stats_btn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>620</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>매출 조회</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
